### ISSUE TITLE
go to 'mouse-pointer' not translated

### ISSUE THREAD

**Issue by chrisgarrity** [LLK/scratch-flash#1043]
Both online and offline editors ignore translating the ' go to' part of the go to 'mouse-pointer' block. Interestingly 'mouse-pointer' is translated.

**Comment by cwillisf** [comment]
This is because the whole string was `go to %m.spriteOrMouse` and is now `go to %m.location`, so Pootle sees it as a brand new string. We had to make this change because we added `random` as an option on that block, but `distance to` and `point towards` still use the old `spriteOrMouse` menu.

I would hope that Pootle would suggest the old `go to %m.spriteOrMouse` translation as "similar" to `go to %m.location` but maybe it's getting tripped up by the fact that "go to" is so short -- over 50% of the characters in the string changed, after all.

In my opinion we should try to insulate the translation layer from the functionality a bit more, so that the translation string can be stable even if implementation details (like the menu name) change. For example, maybe the translation string would be `go to {somewhere}` and there would be a separate piece of data specific to that block which says `somewhere: %m.spriteOrMouse`. We could then change that piece of data to say `somewhere: %m.location` without touching the string tables at all (see: [FTSE](https://en.wikipedia.org/wiki/Fundamental_theorem_of_software_engineering)). This separation is more of a long-term goal, though, and I would want to collect more translator opinions before making such a major change anyway.

For this particular issue, I think I can write a small script to make the change to all languages in one go... I'll give it a shot :)
